searx/engines/crates.py

```diff
@@ -0,0 +1,70 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""Cargo search on crates.io"""
+
+from collections import OrderedDict
+from urllib.parse import urlencode
+
+from dateutil import parser
+
+about = {
+    "website": "https://crates.io/",
+    "wikidata_id": None,
+    "official_api_documentation": "https://crates.io/data-access",
+    "use_official_api": True,
+    "require_api_key": False,
+    "results": "JSON",
+}
+
+categories = ["it", "packages", "cargo"]
+
+
+# engine dependent config
+paging = True
+page_size = 10
```

Comment:
As @Bnyro mentioned .. the page_size is configurable and I reduced from 20 to 10 .. 10 results per page is the default what we have in nearly all engines.